lib/tests/streamlit/config_test.py

```diff
@@ -202,6 +204,7 @@ def test_parsing_toml(self):
             description="This option tests the TOML parser.",
             default_val=DUMMY_VAL_1,
         )
+        config.get_config_options(force_reparse=True)
```

Comment:
It looks like we're doing this get_config_options to just reset the `config_options` essentially?

Reply:
Yeah, an unfortunate side-effect of these changes was that tests are now a bit more annoying to write when they dynamically define new config options (which I don't believe there's a use case for outside of tests) since you need to call `get_config_options(force_reparse=True)` so that those new options start to appear